mantle/platform/api/gcloud/compute.go

```diff
@@ -33,8 +34,56 @@ func (a *API) vmname() string {
 	return fmt.Sprintf("%s-%x", a.options.BaseName, b)
 }
 
+// ["local-ssd:interface=NVME,size=375"]
+func ParseDiskSpec(spec string, zone string) (*compute.AttachedDisk, error) {
```

Comment:
I think it's fine for now to have this here, but we should ideally dedupe this with the one in `qemu.go`. We can have a generic parser that returns the size in gigabytes as a `uint` and any options in a `map[string]string`.

But again, I'm OK to duplicate for now! Maybe let's add a comment in that case?